new-frontiers-14/frontier-station-14
# Medical Tracking Implanter enabled by default

## About the PR
Made it so that by default the medical tracking implant will be selected in loadout

## Why / Balance
Change implemented to help with newer players or players loadouts that may be unmodified before they pick the role by mistake

## How to test

- Make new character
- Check implants tab
- See that medical implanter is selected
- ????
- Profit

## Media

https://github.com/user-attachments/assets/409dcbaf-8d06-4443-9126-29f4dade930c


## Breaking changes
n/a

**Changelog**

:cl:
- add: Medical Trackers are now supplied by default (if left unmodified)
